pom.xml

```diff
@@ -22,7 +22,7 @@
     </scm>
 
     <properties>
-        <otp.serialization.version.id>10</otp.serialization.version.id>
+        <otp.serialization.version.id>11</otp.serialization.version.id>
```

Comment:
This should be changed to 12, currently 11 in dev-2.x.

Reply:
I expect to get a merge conflict on it.

Reply:
Both commits have the same change, so it is automerged